Added view() method to route collections. (#6525)
codeigniter4/CodeIgniter4: user_guide_src/source/changelogs/v4.3.0.rst

======

- Added ``$routes->useSupportedLocalesOnly(true)`` so that the Router returns 404 Not Found if the locale in the URL is not supported in ``Config\App::$supportedLocales``. See :ref:`Localization <localization-in-routes>`

> - Added new ``$routes->view()`` method to echo the view directly. See :ref:`View Routes <view-routes>`.

Reviewer: ```suggestion
- Added new ``$routes->view()`` method to return a view directly. See :ref:`View Routes <view-routes>`.
```
